### PR TITLE
Make disputer an independent sub-package

### PR DESCRIPTION
**Pull request by mrice32**
A few notes:
- The disputer will continue to import packages from across the repo since there's no other way to import those js files.
- I've made the dependencies only include packages that the disputer explicitly requires. All other dependencies will come through cross-dependencies elsewhere in the repo. The only exception to this may be truffle, but it's unclear how we'll run truffle once all the dust settles, so I decided to leave it out for now.
- I kept the package private so we'll avoid publishing it until the entire repo is converted.

Related to #1720.

### CODE REVIEW

File: disputer/package.json [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
-  "name": "disputer",
-  "version": "0.1.0",
+  "name": "@umaprotocol/disputer",
+  "version": "1.0.0",
   "description": "UMA Disputer",
   "private": true,
   "dependencies": {
-    "winston": "^3.2.1",
-    "winston-transport": "^4.3.0"
+    "dotenv": "^6.2.0"
```
Comment (nicholaspai) [inline disputer/package.json:7]: Just want to learn more: shouldn't `dotenv` get installed by the root package.json?

Comment (nicholaspai) [reply disputer/package.json:7]: Resolved IRL: Adding any external imports to this list including `dotenv`, `winston` and `sinon`. All other imports are imported from other directories within this monorepo.

File: disputer/package.json [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
   "description": "UMA Disputer",
   "private": true,
   "dependencies": {
-    "winston": "^3.2.1",
-    "winston-transport": "^4.3.0"
+    "dotenv": "^6.2.0",
+    "sinon": "^9.0.2",
+    "winston": "^3.2.1"
+  },
+  "homepage": "http://umaproject.org",
```
Comment (nicholaspai) [inline disputer/package.json:11]: This should be `https` shouldn't it?

Comment (mrice32) [reply disputer/package.json:11]: Yep, will fix.

Comment (mrice32) [reply disputer/package.json]: Done.

Comment (nicholaspai) [reply disputer/package.json]: LGTM!

**Pull request merged**